i18n/es/user_guide.md

```diff
@@ -30,59 +40,115 @@ Para una lista completa detallada:
 - `kovri.conf` archivo de configuración del router y cliente
 - `tunnels.conf` archivo de configuración para cliente/servidor de túneles
 
-## Paso 3. Iniciar Kovri
+## Paso 4. (Opcional) Configurar tuneles
+
+En resumen, *túneles de cliente* son túneles que usted usa para conectarse a otros servicios y *túneles de servidor* se utilizan para cuando usted aloja servicios (y otras personas se conectan a su servicio).
+
+De forma predeterminada, tendrá túneles de cliente configurados para IRC (Irc2P) y correo electrónico (i2pmail). Para agregar/eliminar túneles de clientes, consulte `tunnels.conf`.
+
+Al crear túneles de servidor, deberá crear *claves privadas persistentes*. Para hacerlo, descomente o cree `keys = your-keys.dat` y reemplace`your-keys` con un nombre apropiado. **No comparta su archivo `.dat` privado con nadie, ¡y asegúrese de hacer una copia de seguridad!**
+
+Una vez configurada, su [dirección Base32](https://getmonero.org/resources/moneropedia/base32-address) se mostrará en su registro después de iniciar kovri. También puede encontrar la dirección en un archivo de texto junto con el archivo de claves privadas en su ruta de datos kovri en el directorio `client/keys`. La dirección dentro de este archivo de texto `.txt` es segura de distribuir para que otras personas puedan conectarse a su servicio.
+
+Ejemplo:
+
+- Archivo de claves privadas: `client/keys/your-keys.dat`
+- Dirección pública [Base32](https://getmonero.org/resources/moneropedia/base32direction)/[Base64](https://getmonero.org/resources/moneropedia/base64-address): `client/keys/your-keys.dat.txt`
+
+**Nota: vea kovri.conf para encontrar su ruta de datos para Linux / OSX / Windows**
+
+## Paso 5. (Opcional) Registre su nuevo [eepsite](https://getmonero.org/resources/moneropedia/eepsite)
+
+**ALTO! Hasta que [#498](https://github.com/monero-project/kovri/issues/498) se resuelva, solo considere registrar su servicio con Kovri y *not* stats.i2p!**
+
+- Abra una solicitud con `[Solicitud de suscripción] your-host.i2p` (reemplace su-host.i2p con su nombre de host deseado) en el [issue tracker de Kovri](https://github.com/monero-project/kovri/issues)
+- En el cuerpo de la solicitud, pegue los contenidos de su archivo público `.txt` que se mencionó en el paso anterior
+- Después de la revisión, agregaremos su host y firmaremos la suscripción
+- ¡Hecho!
+
+## Paso 6. Ejecutar Kovri
 ```bash
-$ cd build/ && ./kovri
+$ cd build / && ./kovri
```

Comment:
Won't work. Keep the old line.

Reply:
Done, don't know how it slip, since i didn't touch the code.